pkg/releases/releases_test.go

```diff
@@ -205,6 +213,12 @@ func TestReleases_ReversableSerializationMultiple_Success(t *testing.T) {
 			},
 		},
 		LanguagesGenerated: map[string]releases.GenerationInfo{},
+		LanguageChangelog: map[string]string{
+			"go":     "## Go SDK Changes Detected:\n* sdk.UserService.GetProfile(): **Added**\n* sdk.UserService.DeleteAccount(): **Deleted**\n* sdk.UserService.UpdateSettings(): **Changed** request schema\n* sdk.AdminService.GenerateReport(): **Added** request.reportType [breaking]\n",
```

Comment:
I think this doesn't really make sense to be a repo-specific file instead of an SDK-specific file. Someone consuming a library doesn't really care for the changes in (for instance) Python if they're consuming NodeJS.

I.e. I think we want a RELEASES.md to be *inside* an SDK, not outside of it, as we want it included in (for instance) pypi, npm etc.